pkgs/darwin-installer/default.nix

```diff
@@ -1,10 +1,21 @@
 { stdenv, writeScript, nix, pkgs }:
 
 let
+  configuration = builtins.path {
+    name = "nix-darwin-installer-configuration";
+    path = ./.;
+    filter = name: _type: name != toString ./default.nix;
+  };
+
+  darwin = builtins.path {
+    name = "nix-darwin";
+    path = ../..;
```

Comment:
Any reason not to filter out .git? This repo isn't particularly big but that does make this pretty unreproducible.

Reply:
Looks good, thanks!